regulations/templates/regulations/preamble-partial.html

```diff
@@ -17,12 +17,11 @@
             <div class="queued"></div>
             <input type="file">
             <button type="submit">Save</button>
-            {# <div class="comment-clear">Clear this comment</div> #}
```

Comment:
Just to verify, we don't want this functionality anymore (preferring to blank out + delete files instead)?